src/repositories/projectRepository.ts

```diff
@@ -28,6 +38,85 @@ export const findProjectById = (
     .getOne();
 };
 
+// return query without execution
```

Comment:
Moved Filter query to the repository so I could reuse it for count query and results query.
When using the same querybuilder object, the cache was throwing errors.